core/core-impl/src/main/java/ai/timefold/solver/core/impl/util/Pair.java

```diff
@@ -8,6 +10,35 @@
  * @param <Key_>
  * @param <Value_>
  */
-public record Pair<Key_, Value_>(Key_ key, Value_ value) {
+public class Pair<Key_, Value_> {
```

Comment:
This causes Javadoc plugin to fail. The params need to be on the constructor, not on the class. (As it's no longer a record with a canonical constructor.)